docs/source/tutorial.rst

```diff
@@ -7,47 +7,395 @@ Tutorial
+    temp_path = my_downloader(melody_path)
+
+    # Accessing to track melody annotation
+    example_melody = orchset.load_melody(temp_path)
+
+    print(example_melody.frequencies)
+    >>> array([  0.   ,   0.   ,   0.   , ..., 391.995, 391.995, 391.995])
+    print(example_melody.times)
+    >>> array([0.000e+00, 1.000e-02, 2.000e-02, ..., 1.244e+01, 1.245e+01, 1.246e+01])
+
+
+
+Annotation classes
+^^^^^^^^^^^^^^^^^^
+
+``mirdata`` uses several diffent data classes to store annotations. These data classes are meant to standarize the organization for
+all the loaders, and keep compatibility with `JAMS <https://jams.readthedocs.io/en/stable/>`_ and `mir_eval <https://craffel.github.io/mir_eval/>`_.
+
+The list and descriptions of available annotation classes can be found `in this link <https://mirdata.readthedocs.io/en/latest/source/mirdata.html#module-mirdata.annotations>`_.
+
+**These classes are extendable in case a certain loader requires it.**
 
 Iterating over datasets and annotations
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In general, most datasets are a collection of tracks. Each track has an audio with its own annotations.
+
+With the ``load_tracks()`` method, all the tracks (so including their respective audio and annotations) can be loaded
+in a dictionary with the ids as keys and tracks as items.
+
+.. code-block:: python
+
+    orchset = mirdata.initialize('orchset')
+    for key, track in orchset.load_tracks().items():
+        print(key, track.title, track.audio_path)
+
+
+Alternatively, we can run over the ``track_ids`` list to access directly to each track in the dataset.
 
+.. code-block:: python
 
-Working with remote datasets
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    orchset = mirdata.initialize('orchset')
+    for track_id in orchset.track_ids:
+        print(track_id, orchset.track(track_id).title,  orchset.track(track_id).audio_path)
+
+
+Working with remote index
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For the end user there is no difference between the remote and local indexes.
+
+However, to extend the library you have to add in utils.LargeData(...) the remote_index argument with a
+download_utils.RemoteFileMetadata dictionary with the remote index information.
+
+DATA = utils.LargeData("acousticbrainz_genre_index.json", remote_index=REMOTE_INDEX)
+
+.. code-block:: python
+
+    REMOTE_INDEX = {
+        "REMOTE_INDEX": download_utils.RemoteFileMetadata(
+            filename="acousticbrainz_genre_index.json.zip",
+            url="https://zenodo.org/record/4298580/files/acousticbrainz_genre_index.json.zip?download=1",
+            checksum="810f1c003f53cbe58002ba96e6d4d138",
+            destination_dir="",
+        )
+    }
+    DATA = utils.LargeData("acousticbrainz_genre_index.json", remote_index=REMOTE_INDEX)
 
 
 Working with big datasets
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
+In the development of large datasets, it is advisable to create an index as small as possible to develop
+the new dataset and pass the tests.
```

Comment:
This feels a comment for `contributing`, maybe move it there? The idea of having this section here was to explain the use of remote data, but it is addressed in `Accessing annotations remotely`

Reply:
Yes! That's is a good idea. I moved it into the testing section and I also added a reminder to keep the testing resources as light as possible!